code/modules/mob/living/silicon/robot/robot.dm

```diff
@@ -373,7 +376,7 @@
 
 	var/picked  = pick(module_sprites)
 	icon_state = module_sprites[picked]
-		
+
```

Comment:
Actually what editor do you use? If you use Atom it does this by default.

Sorry but I have to close your PR now.